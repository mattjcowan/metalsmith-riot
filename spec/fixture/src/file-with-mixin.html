---
type: test
---
<!DOCTYPE html>
<html dir="ltr" lang="en-US">
<head></head>
<body>
Sample mixin in metalsmith:

<mixin-sample></mixin-sample>

</body>
</html>