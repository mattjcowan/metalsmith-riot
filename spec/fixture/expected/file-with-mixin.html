<!DOCTYPE html>
<html dir="ltr" lang="en-US">
  <head></head>
  <body>
    Sample mixin in metalsmith:
    <mixin-sample><span>From file metadata: test</span> <span>From global metadata: someValue</span></mixin-sample>
  </body>
</html>